src/k8s/request.js

```diff
@@ -0,0 +1,115 @@
+import { API_VERSION, VM_KIND, OS_LABEL, FLAVOR_LABEL} from '../constants';
+
+export const createVM = (basicSettings, network, storage) => {
+  let vm = {
+    apiVersion: API_VERSION,
+    kind: VM_KIND,
+    metadata : {
+      name: basicSettings.name.value,
+      labels: {
+        [OS_LABEL]: basicSettings.operatingSystem.value
+      }
+    },
+      spec: {
+        template: {}
+      }
+  }
+  if(basicSettings.namespace){
+    vm.metadata.namespace = basicSettings.namespace.value;
+  }
+  if(basicSettings.description){
+    vm.metadata.labels.description = basicSettings.description;
+  }
+  addFlavor(vm, basicSettings);
```

Comment:
I think few newlines here and there add to the readability, but that is just my opinion.

Reply:
added a few new lines here and there :)